operations/jsonnet/microservices/configmap.libsonnet

```diff
@@ -109,8 +109,13 @@
 
   tempo_query_frontend_config:: $.tempo_config {},
   tempo_block_builder_config:: $.tempo_config {},
+  tempo_live_store_config:: $.tempo_config {},
   tempo_backend_scheduler_config:: $.tempo_config {},
-  tempo_backend_worker_config:: $.tempo_config {},
+  tempo_backend_worker_config:: $.tempo_config {
+    backend_worker+: {
+      backend_scheduler_addr: 'backend-scheduler.%s.svc.cluster.local.:9095' % [$._config.namespace],
```

Comment:
Noticed this was missing, but the `backend_worker` can't start without this. Is it okay to include by default @zalegrala? It follows the same pattern of the query-frontend / querier.

Reply:
Looks good to me. :+1: